openapi/p4pa-workflow-hub.openapi.yaml

```
@@ -229,6 +229,33 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/WorkflowErrorDTO'
+  /workflow/export-file/{exportFileId}:
+    get:
+      tags:
+        - ExportFile
+      operationId: exportFileExpirationHandler
+      description: Handle the expiration of export file
```

move it after exportFile operation
```suggestion
  /export-file/{exportFileId}/expire:
    post:
      tags:
        - ExportFile
      operationId: expireExportFile
      description: Expire an export file
```

---

move it after exportFile operation